CMakeLists.txt

```diff
@@ -183,8 +186,9 @@ if(NOT (DJINNI_WITH_OBJC OR DJINNI_WITH_JNI OR DJINNI_WITH_PYTHON OR DJINNI_WITH
   message(FATAL_ERROR "At least one of DJINNI_WITH_OBJC or DJINNI_WITH_JNI or DJINNI_WITH_PYTHON or DJINNI_WITH_CPPCLI must be enabled.")
 endif()
 
+option(DJINNI_BUILD_TESTING "Build tests" ON)
 
 include(CTest)
-if (BUILD_TESTING)
+if (BUILD_TESTING AND DJINNI_BUILD_TESTING)
```

Comment:
`BUILD_TESTING` is a global cmake variable that will affect the whole project. If the support-lib is installed by embedding the sources in cmake, setting the variable could potentially disable the tests for the parent project itself, too. In this case the new option `DJINNI_BUILD_TESTING` can be used to disable tests just for the support-lib.

Reply:
you created answered while I wrote the question, that's great!